ephios/extra/management/commands/setupdata.py

```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
+            uuid=uuid.UUID("a3c5a3a0-f577-4bf5-8b2f-84108022d793"),
+        )
+        tauchf.included_qualifications.add(taucher_2)
+
+        boats_category = QualificationCategory.objects.create(
+            title=_("Boats (DLRG)"),
+            uuid=uuid.UUID("f1e2619b-56ee-4e4a-b429-b923f8586771"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein Binnen (DLRG)",
+            abbreviation="Bootsführerschein Binnen",
+            uuid=uuid.UUID("7ec0c9e2-f211-4370-b796-e309f3e9b448"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein See (DLRG)",
+            abbreviation="Bootsführerschein See",
+            uuid=uuid.UUID("1ba25ee1-fa88-4d44-9416-3a024caa6daf"),
+        )
+
+        wr_category = QualificationCategory.objects.create(
+            title=_("Water rescue (DLRG)"),
+            uuid=uuid.UUID("93574a61-1a6e-4dd5-8df3-40e8c51b693f"),
+        )
+        rettschw = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer",
+            abbreviation="Rettungsschwimmer",
+            uuid=uuid.UUID("ef95a854-2eeb-431c-a795-bc291b341d49"),
+        )
+        wr = Qualification.objects.create(
+            category=wr_category,
+            title="Fachausbildung Wasserrettungsdienst (DLRG)",
+            abbreviation="Wasserretter",
+            uuid=uuid.UUID("bd7ca398-ed2a-4e97-b681-bc8fb1138ada"),
+        )
+        wr.included_qualifications.add(rettschw)
+        sr1 = Qualification.objects.create(
+            category=wr_category,
+            title="Strömungsretter I (DLRG)",
+            abbreviation="Strömungsretter I",
+            uuid=uuid.UUID("e1c1fac1-146d-4a52-935e-36455bcf0f87"),
+        )
+        sr1.included_qualifications.add(wr)
+        sr2 = Qualification.objects.create(
+            category=wr_category,
+            title="Strömungsretter II (DLRG)",
+            abbreviation="Strömungsretter II",
+            uuid=uuid.UUID("9c506e7e-a456-4f22-8027-5365ab7dc58c"),
+        )
+        sr2.included_qualifications.add(sr1)
+
+        radio_category = QualificationCategory.objects.create(
+            title=_("Radio"),
+            uuid=uuid.UUID("368fb612-f8c5-476a-a4e7-1478996a8ab3"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Digitalfunk",
+            abbreviation="Digitalfunk",
+            uuid=uuid.UUID("506c4d4c-df11-4d41-95d7-c4ddfc78b706"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Analogfunk",
+            abbreviation="Analogfunk",
+            uuid=uuid.UUID("4974ea66-2086-4987-958b-503de21a285a"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Betriebsfunk (DLRG)",
```

Comment:
The qualification is called "DLRG-Sprechfunker". All other entries are named according to a qualification, this one should be, too.